src/css/components/panel.scss

```diff
@@ -68,61 +69,44 @@ h4 + .panel {
 .panel-group {
   @include clearfix;
 
-  margin-bottom: 0.65rem;
-
   & + .panel-group {
     margin-top: $grid-4;
   }
 }
 
 .panel-actions {
-  > * {
-    margin-right: 2rem;
+  margin-top: $grid-4;
 
-    &:last-child {
-      margin-right: 0;
-    }
+  .action {
```

Comment:
Oops, nope. I tried this and the negative margin was great for the actions, but broke adjacent columns in a way I couldn't fix.